### DISCUSSION TITLE
Install on Windows

### DISCUSSION THREAD

**Discussion by IanIreland** [celsworth/lxp-bridge#162] [General]
I would love to install the bridge and get it running on Windows. I have never used Docker and have tried unsuccessfully.
I have MQTT working successfully as part of my HA system, feeding into HomeSeer.
Can anybody point me in the direction of an 'idiot's guide' to getting it working on Windows?

**Comment by JohnMacrae** [comment]
Ian, there are many videos on this topic on Youtube. I'd suggest you take a
look and see how you get on. I speak as someone who did the same and it was
pretty easy. However, my installation faltered when it came to setting up
the networking to address my MQTT server which is on a different machine.

Good luck - tell us what you find and we can perhaps add a Wiki page for it
to help others in the future.

Kind Regards
John

**Reply by IanIreland** [reply]
I only see Youtubes for setting up Docker. I have Docker for Desktop setup and running. I'm struggling knowing where to put the YAML file for the lxp-bridge container.

**Reply by celsworth** [reply]
You need to bind-mount the  YAML into the container via the `--mount` flag to Docker, like this:

```
--mount type=bind,source=C:/lxp-bridge/config.yaml,target=/etc/config.yaml
```

(note I've not actually used Docker on Windows, but it seems that you use forward slashes after `C:` from a bit of Googling)

So where it actually lives doesn't matter providing you specify that path when starting the container. There might be ways to do this in the GUI as well. I'll install Docker for Windows to have a play with it.
